README.md

```diff
@@ -1,2 +1,8 @@
-# spog-test
-Automation tests for the SPOG UI
+# SPoG-test
+Automation tests for the SPoG UI
+
+This repo deals with UI automation tests for the UI part of the Trustification product.
+The infrastructure is Playwright and the language used is Python.
+Along the way there will be attempts to add these tests to the Trustification repo (https://github.com/trustification/trustification)
```

Comment:
Hey @ikanias, you think this is really a possibility? I thought the UI tests would live in this repo, and there would only be API level tests in the Trustification repo?

Reply:
The UI tests will remain here in this repo. Jens proposed that from time to time we will have some kind of a trigger that will send them our PRs so they can test it with their code. Just to make sure nothing breaks.